frontend/doc-site/README.mdx

```diff
@@ -0,0 +1,36 @@
+# cellxgene
+
+![cellxgene explorer](.gitbook/assets/image (12).png)
+
+## Introduction to cellxgene
+
+Cellxgene enables scientists to annotate, publish, find, download, explore and analyze single cell datasets. Cellxgene has two offerings (which both give you access to the [cellxgene explorer](explore-data/explorer-tutorials.md)):
+
+* The [**cellxgene** **Data Portal**](https://cellxgene.cziscience.com) - a Software-as-a-Service (SaaS) platform to publish, find, download, and explore single cell data. Enables you to explore yours and other authors' publication ready data.
```

Comment:
should we include archive or store to indicate that this is meant to be a permanent repository for the data?